src/etc/tidy.py

```diff
@@ -24,6 +24,7 @@
 
 interesting_files = ['.rs', '.py', '.js', '.sh', '.c', '.h']
 uninteresting_files = ['miniz.c', 'jquery', 'rust_android_dummy']
+stable_whitelist = []
```

Comment:
I fixed it in my latest commit. Apparently the real culprit is the `re.match` down at `if fileinput.isFirstLine()` - what I really wanted was `re.search()` instead.